.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "python.pythonPath": "C:\\Python38\\python.exe"
+    "python.pythonPath": ".venv\\Scripts\\python.exe"
 }
```

Comment:
This file should be in `.gitignore` because it may vary on the user.

Reply:
Added to .gitignore